CoreErrorReportView/release/modules/ext/docs/CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/error-report.md

```diff
@@ -0,0 +1,56 @@
+# Error Report
```

Comment:
might be nice if you could add a little explanation as to the purpose of the view. and where appropriate, some descriptions next to the dropdown options providing more detail as to what they mean